pkg/services/ngalert/api/api_alertmanager.go

```diff
@@ -125,6 +126,16 @@ func (srv AlertmanagerSrv) RouteGetAMStatus(c *models.ReqContext) response.Respo
 }
 
 func (srv AlertmanagerSrv) RouteCreateSilence(c *models.ReqContext, postableSilence apimodels.PostableSilence) response.Response {
+	if !c.HasUserRole(models.ROLE_EDITOR) {
+		return ErrResp(http.StatusForbidden, errors.New("permission denied"), "")
+	}
```

Comment:
This is not necessary anymore as it is covered in [authorization code](https://github.com/grafana/grafana/blob/c1dbe7617ce98e581de4ab89aadd4639755079f5/pkg/services/ngalert/api/authorization.go#L33)